content/en/docs/setup/install/istioctl/index.md

```diff
@@ -362,26 +362,6 @@ spec:
 $ istioctl install -f telemetry_off.yaml
 {{< /text >}}
 
-Another customization is to select different namespaces for features and components. The following is an example
```

Comment:
Instead of completely removing this section, should we change the example to something else that illustrates the same feature (components different namespaces), e.g., an ingress-gateway in a different namespace?

cc @ostromart

Reply:
I'm actually in favor of removing. Although theoretically different namespaces can be selected, in practice using the same namespace is what's recommended and tested. In fact we may want to consider deprecating this part of the API and use only the global namespace (gateways are an exception).

Reply:
Wow, and I remember when we said it was better to have everything in its own namespace (and not that long ago).